jaseci_kit/jaseci_kit/encoders/utils/tokenizer.py

```diff
@@ -31,17 +31,23 @@ def __init__(self, contexts, candidates, context_transform,
                 'labels': []
             }
             for text, cand, lbl in zip(contexts, candidates, labels):
-                if lbl == 1 and len(group['candidates']) > 0:
-                    self.data_source.append(group)
-                    group = {
-                        'context': None,
-                        'candidates': [],
-                        'labels': []
-                    }
+                group = {
+                    'context': None,
+                    'candidates': [],
+                    'labels': []
+                }
+                # below code is to combine positive and negetive sample
```

Comment:
Same here.

Reply:
this code was used when, the positive and negative sample for 1 datapoint is combined